src/basic/missing_syscall.h

```diff
@@ -78,7 +78,7 @@ static inline int missing_memfd_create(const char *name, unsigned int flags) {
 /* ======================================================================= */
 
 #if !HAVE_GETRANDOM
-static inline int missing_getrandom(void *buffer, size_t count, unsigned flags) {
+static inline ssize_t missing_getrandom(void *buffer, size_t count, unsigned flags) {
```

Comment:
Maybe, it is better to add some comments?

Reply:
I'm not certain what kind of comment you are looking for, but I added a note about matching glibc's definition.